contracts/bridge/contracts/binance/LightClientBridge.sol

```diff
@@ -439,11 +475,6 @@ contract LightClientBridge is Bitfield, ValidatorRegistry, GuardRegistry {
         // Encode and hash the commitment
         bytes32 commitmentHash = createCommitmentHash(commitment);
 
-        require(
-            commitmentHash == data.commitmentHash,
-            "Bridge: Commitment must match commitment hash"
-        );
-
```

Comment:
We can not delete this check, we need it.

Reply:
Fix